Zend/tests/attributes/002_rfcexample.phpt

```diff
@@ -0,0 +1,41 @@
+--TEST--
+Attributes: RFC Example
+--FILE--
+<?php
+namespace My\Attributes {
+    use PhpAttribute;
+
+    <<PhpAttribute>>
+    class SingleArgument {
+        public $argumentValue;
+
+        public function __construct($argumentValue) {
+             $this->argumentValue = $argumentValue;
+        }
+    }
+}
+
+namespace {
+    use My\Attributes\SingleArgument;
+
+    <<SingleArgument("Hello World")>>
+    class Foo {
+    }
+
+    $reflectionClass = new \ReflectionClass(Foo::class);
+    $attributes = $reflectionClass->getAttributes();
+
+    var_dump($attributes[0]->getName());
+    var_dump($attributes[0]->getArguments());
+    var_dump($attributes[0]->getAsObject());
```

Comment:
Something I didn't see in the patch: what happens if something marked with `<<PhpAttribute>>` does not have a constructor, or has a `private` constructor?

Reply:
Errata: covered by scenarios below :+1: